app/pb_kits/playbook/pb_circle_icon_button/_circle_icon_button.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import classnames from "classnames"
+import {Button, Icon} from "../"
+
+type CircleIconButtonProps = {
+  variant?: 'primary' | 'secondary' | 'link',
+  disabled?: Boolean,
+  dark?: Boolean,
+  className?: String,
+  icon: String,
+  id?: String,
+};
+
+const circleIconButtonCSS = ({
+  variant='primary',
+  disabled=false,
+  dark=false,
+}: circleIconButtonProps) => {
+  const variantStyle = variant !== null ? `_${variant}` : ''
+  const disabledStyle = disabled === true ? '_disabled' : ''
+  const darkStyle = dark === true ? '_dark' : ''
+  return 'pb_circle_icon_button_kit' + variantStyle + disabledStyle + darkStyle
+}
+
+const CircleIconButton = (props: CircleIconButtonProps) => {
+  const {
+    variant,
+    disabled,
+    icon,
+    dark
+  } = props
+
+  return (
+    <div className={'pb_circle_icon_button_kit'}>
+      <Button text={null} variant={variant} disabled={disabled} dark={dark}>
+        <Icon fixedWidth icon={icon} />
```

Comment:
Once https://github.com/powerhome/playbook/pull/466 is merged, you can use the `<IconCircle/>` component here